app/transformers/api/v4/teleconsultation_medical_officer_transformer.rb

```diff
@@ -1,6 +1,7 @@
 class Api::V4::TeleconsultationMedicalOfficerTransformer
   class << self
-    def to_response(medical_officer)
+    include Memery
+    memoize def to_response(medical_officer)
```

Comment:
I think instead of doing this we could just retrieve distinct medical_officers, no?

Reply:
We do in fact retrieve distinct MOs from the DB. The API wants to return a record for every MO<>Facility pair, and we end up doing this transformation multiple (100s of) times. The bulk of the time is spent in application code as per DD as well.